editoast/src/views/rolling_stocks/mod.rs

```diff
@@ -124,6 +124,30 @@ impl RollingStockForm {
     }
 }
 
+#[derive(QueryableByName, Debug, Clone, Serialize, Deserialize)]
+struct PowerRestrictions {
+    #[diesel(sql_type = SqlText)]
+    power_restrictions: String,
+}
```

Comment:
A tiny bit of nitpicking

```suggestion
struct PowerRestriction {
    #[diesel(sql_type = SqlText)]
    power_restriction: String,
}
```